examples/integrations/uniswap/test_funcs.py

```diff
@@ -0,0 +1,104 @@
+import os
+
+from ape import Contract, accounts, networks
+from dotenv import find_dotenv, load_dotenv
+
+from giza_actions.integrations.uniswap.uniswap import Uniswap
+
+networks.parse_network_choice(f"ethereum:mainnet-fork:foundry").__enter__()
+load_dotenv(find_dotenv())
+
+dev_passphrase = os.environ.get("DEV_PASSPHRASE")
+sender = accounts.load("dev")
+sender.set_autosign(True, passphrase=dev_passphrase)
+
+weth_address = "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"
+usdc_address = "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48"
+token0 = Contract(weth_address)
+token1 = Contract(usdc_address)
+fee = 500
+
+### Uniswap ###
+uni = Uniswap(sender=sender, version=3)
+pool = uni.get_pool(token0, token1, fee)
+price = pool.get_pool_price(invert=True)
+print(f"--------- Pool Price: {price}")
+
+### Quoter ###
+amount_in = int(1e8)
+# calling using pool object
+amount_out = uni.quoter.quote_exact_input_single(amount_in, pool=pool)
```

Comment:
I feel like using `uni.quoter` or `uni.nft_manager` everytime that we want to execute makes the `Uniswap` class a little bit useless as we could just do the init by ourselves of the other classes and call directly de quoter or router.

Maybe we need to think about how we organize the code